Doesn’t build on ruby 2.5.0

Builds fine on ruby 2.4.2, but 2.5.0 throws an error not knowing what `ID` is, so you’re probably missing a header you should be directly requiring.

I had this as well.

```
current directory: ${GEM_HOME}/gems/rb-scpt-1.0.2
make "DESTDIR="
compiling src/SendThreadSafe.c
compiling src/rbae.c
In file included from src/rbae.c:12:
In file included from src/osx_ruby.h:2546:
/usr/local/Cellar/ruby/2.5.0_2/include/ruby-2.5.0/ruby/backward.h:32:36: warning: type specifier missing, defaults to 'int' [-Wimplicit-int]
DEPRECATED(void rb_autoload(VALUE, ID, const char*));
                                   ^
/usr/local/Cellar/ruby/2.5.0_2/include/ruby-2.5.0/ruby/backward.h:53:45: error: unknown type name 'ID'
DEPRECATED(int rb_frame_method_id_and_class(ID *idp, VALUE *klassp));
```

To work around I commented out the line in `backward.h` and then uncommented it once the Gem was built.

--- a/extconf.rb
+++ b/extconf.rb
@@ -53,8 +53,6 @@ new_filename_prefix = 'osx_'
   $stderr.puts "create #{File.expand_path(dst_fname)} ..."
   File.open(dst_fname, 'w') do |dstfile|
     IO.foreach(src_path) do |line|
-      line = line.gsub(/\bID\b/, 'RB_ID')
-      line = line.gsub(/\bT_DATA\b/, 'RB_T_DATA')
       line = line.gsub(/\b(?:ruby\/)?intern.h\b/, "#{new_filename_prefix}intern.h")
       line = line.gsub('#include "defines.h"', '#include "ruby/defines.h"') if not is_ruby_18
       dstfile.puts line

